docker_xylem/service.py

```diff
@@ -28,9 +28,14 @@ def __init__(self, config):
         self.xylem_host = config['host']
         self.xylem_port = config.get('port', 7701)
         self.mount_path = config.get('mount_path', '/var/lib/docker/volumes')
+        self.volume_name = config.get('volume_name', 'seed')
+
+        self.volume_path = os.path.join(self.mount_path, self.volume_name)
 
         self.current = {}
```

Comment:
Hmmm, we probably should find a better way to provide the List API request, that might actually be why docker decides stuff is 'local' because it doesn't appear in list_volumes after service restarts.